### ISSUE TITLE
Request for a New Dashboard UI

### ISSUE THREAD

**Issue by MAVERICK-VF142** [MAVERICK-VF142/Drx.MediMate#6]
A new dashboard UI is needed to improve the user experience and provide better visualization and accessibility. The current dashboard does not meet our requirements in terms of design, usability, or features. Please design and implement a modern, responsive dashboard interface with enhanced functionality and an updated look and feel.

**Comment by Vasita27** [comment]
Hello, can i work on this

**Comment by MAVERICK-VF142** [comment]
@Vasita27 please give progress update

**Comment by MAVERICK-VF142** [comment]
thankyour for update @Vasita27

**Comment by Vasita27** [comment]
Hi, I think the page changed before i can create a pr, so it is showing conflicts.
